packages/ag-charts-community/src/chart/marker/marker.ts

```diff
@@ -15,9 +18,9 @@ export abstract class Marker extends Path {
 
     override computeBBox(): BBox {
         const { x, y, size } = this;
-        const half = size / 2;
+        const { center } = this.constructor as any;
```

Comment:
This seems a bit of a weird way to do this - could you either just declare a non-static `center` (and assign a constant value), allowing the sub-class to update this property, or add a `getCenter(): Point` method that sub-classes can override?

Reply:
The reason for this was otherwise we'd have to pass actual marker instances to `labelPlacement` - but they wouldn't actually correspond with real markers in the scene (and wouldn't have their x/y set). We needed a way to get the centre without actually constructing the instances

Reply:
Ok, let's roll with this for now.